examples/features/broker-connection/amqp-sending-overssl/src/main/resources/activemq/server0/broker.xml

```diff
@@ -37,7 +37,7 @@ under the License.
 
       <broker-connections>
          <amqp-connection uri="tcp://localhost:5772?sslEnabled=true;trustStorePath=activemq.example.truststore;trustStorePassword=activemqexample" name="otherSSL" retry-interval="1000">
-            <sender match="#"/>
+            <sender address-match="#"/>
```

Comment:
#3321 may conflict here, can you please jsut update it and merge, ill be heading off now.